app/javascript/components/molecules/DropdownMenu.js

```diff
@@ -2,41 +2,37 @@ import React from 'react'
 import DropdownMenuItem from '../atoms/DropdownMenuItem'
 import formUtils from '~/utils/formUtils'
 
-class DropdownMenu extends React.Component {
-  handleOnChange = (e, value, label) => this.props.onChange && this.props.onChange(value, label)
+const DropdownMenu = ({ items, value, style, classes, onChange }) => {
+  const handleOnChange = (e, val, label) => onChange && onChange(val, label)
 
-  handleOnKeyDown = (e, value, label) => {
+  const handleOnKeyDown = (e, val, label) => {
```

Comment:
I'm not super clear on what this functionality with keyboards was for dropdowns before, do you?

Reply:
These keycodes are `enter` and `space` so I assume that this was added so that you can select an item without clicking it with the mouse